src/docs/meta/relay/best-practices.mdx

```diff
@@ -32,8 +92,10 @@ This example configuration contains values you can adjust to change both Relay's
 ```yaml
 ---
 relay:
-  upstream: https://o<YOUR_ORG_ID>.ingest.sentry.io
+  upstream: https://sentry.io.
```

Comment:
What if we leave this on the ingest domain and instead describe where to get this? I'm thinking we can add this URL to the Relay settings page. cc @matejminar 

Otherwise, Relays circumvent org domain protection.

Reply:
Reverted.